src/Adyen/HttpClient/CurlClient.php

```diff
@@ -273,22 +273,22 @@ protected function handleCurlError($url, $errno, $message, $logger)
     {
         switch ($errno) {
             case CURLE_OK:
-                $msg = "Probably your Web Service username and/or password is incorrect";
+                $msg = 'Probably your Web Service username and/or password is incorrect';
                 break;
             case CURLE_COULDNT_RESOLVE_HOST:
             case CURLE_OPERATION_TIMEOUTED:
                 $msg = "Could not connect to Adyen ($url).  Please check your "
-                    . "internet connection and try again.";
+                    . 'internet connection and try again.';
                 break;
             case CURLE_SSL_CACERT:
             case CURLE_SSL_PEER_CERTIFICATE:
                 $msg = "Could not verify Adyen's SSL certificate.  Please make sure "
-                    . "that your network is not intercepting certificates.  "
+                    . 'that your network is not intercepting certificates.  '
                     . "(Try going to $url in your browser.)  "
-                    . "If this problem persists,";
```

Comment:
Seems a bit strange to have different types of quotes for the same concatenated string

Reply:
Variable do not extend in single quotes.
The default is the single quote. Double quotes are for

1. If the string contains a single quote
2. If there is a variable in it

I leave it up to you!

Quote from the top comment:

> Done by a machine! (phpcbf)

Reply:
To be clear.
I'm not willing to contribute to quote thingies.